iogt/static/css/modules/_m-pagination.scss

```diff
@@ -9,5 +9,5 @@
 
 .pagination__link--current {
 	font-size: $font-x-small;
-	padding: 15px 15px 0;
+	padding: 15px 15px 0 0;
```

Comment:
Are these changes meant for the smartphone designs, because you're changing them in the feature phone files?